src/VisualStudio/CSharp/Impl/EditorConfigSettings/DataProvider/CodeStyle/CSharpCodeStyleSettingsProvider.cs

```diff
@@ -151,6 +151,7 @@ private IEnumerable<CodeStyleSetting> GetExpressionCodeStyleOptions(AnalyzerConf
             yield return CodeStyleSetting.Create(CSharpCodeStyleOptions.PreferRangeOperator, description: ServicesVSResources.Prefer_range_operator, editorConfigOptions, visualStudioOptions, updaterService, FileName);
             yield return CodeStyleSetting.Create(CSharpCodeStyleOptions.ImplicitObjectCreationWhenTypeIsApparent, description: CSharpVSResources.Prefer_implicit_object_creation_when_type_is_apparent, editorConfigOptions, visualStudioOptions, updaterService, FileName);
             yield return CodeStyleSetting.Create(CSharpCodeStyleOptions.PreferTupleSwap, description: ServicesVSResources.Prefer_tuple_swap, editorConfigOptions, visualStudioOptions, updaterService, FileName);
+            yield return CodeStyleSetting.Create(CSharpCodeStyleOptions.PreferTrailingCommas, description: ServicesVSResources.Prefer_trailing_commas, editorConfigOptions, visualStudioOptions, updaterService, FileName);
```

Comment:
hrmm... i don't think this is an expression-style...   This is all for declarations (or maybe variables).  consider putting in the variable section?  or making a new section?

Reply:
@CyrusNajmabadi Actually even those in `GetVariableCodeStyleOptions` have `CSharpCodeStyleOptionGroups.ExpressionLevelPreferences`. I'm not sure what's the suitable group here.